app/src/integrationTests/ReportIntegrationTests.tsx

```diff
@@ -90,13 +82,6 @@ class ReportIntegrationTests extends IntegrationTestSuite {
             expect(versionDetails.name).is.equal("minimal");
         });
 
-        it("fetches report one time token", async () => {
-            const versions = await (new ReportsService(this.store.dispatch, this.store.getState)).getReportVersions("minimal");
```

Comment:
This was testing the old `altjs` source; the same test for the new service is directly below.